bot/resources/tags/empty-json.md

```diff
@@ -0,0 +1,27 @@
+When creating a new JSON file you might run into the following error.
+
+`JSONDecodeError: Expecting value: line 1 column 1 (char 0)`
+
+In short this error means your JSON is invalid in it's current state.
+A JSON may never be completely empty and must always at least have one of the following items.
+
+```
+object
+array
+string
+number
+"true"
```

Comment:
Aren't `true`, `false`, and `null` valid literals in a JSON? If so, why are they in quotes here? It might be better to have an example.

Reply:
You might be looking at an old commit. In the latest one they are removed. It's purely list and object now.